src/components/input/input.test.jsx

```diff
@@ -16,314 +16,311 @@ describe('Input', () => {
+      render((
         <Input
           id="foo"
-          label="I am a label for accessibility"
+          label="the label"
           invalid
         />
-      )).root;
-      expect(tree.findByType('svg').props.className.split(' ')).toContain('invalid-icon');
+      ));
+      expect(screen.getByRole('img')).toHaveClass('invalid-icon');
     });
   });
 
   describe('maxLength API', () => {
     it('sets the maxLength attribute', () => {
-      const tree = renderer.create((
+      render((
         <Input
           id="foo"
-          label="I am a label for accessibility"
+          label="the label"
           maxLength={100}
         />
-      )).root;
-      expect(tree.findByType('input').props.maxLength).toEqual(100);
+      ));
+      expect(screen.getByLabelText('the label')).toHaveAttribute('maxlength', '100');
     });
   });
 
   describe('name API', () => {
     it('sets the name attribute', () => {
-      const tree = renderer.create((
+      render((
         <Input
           id="foo"
-          label="I am a label for accessibility"
+          label="the label"
           name="test-name"
         />
-      )).root;
-      expect(tree.findByType('input').props.name).toEqual('test-name');
+      ));
+      expect(screen.getByLabelText('the label')).toHaveAttribute('name', 'test-name');
     });
   });
 
   describe('onBlur API', () => {
     it('sets the onblur handler', () => {
       const onBlurSpy = jest.fn();
-      const tree = renderer.create((
+      render((
         <Input
           id="foo"
-          label="I am a label for accessibility"
+          label="the label"
           onBlur={onBlurSpy}
         />
-      )).root;
-      tree.findByType('input').props.onBlur();
+      ));
+      fireEvent.blur(screen.getByLabelText('the label'));
       expect(onBlurSpy.mock.calls.length).toEqual(1);
     });
   });
 
   describe('onChange API', () => {
     it('sets the onchange handler', () => {
       const onChangeSpy = jest.fn();
-      const tree = renderer.create((
+      render((
         <Input
           id="foo"
-          label="I am a label for accessibility"
+          label="the label"
           onChange={onChangeSpy}
         />
-      )).root;
-      tree.findByType('input').props.onChange();
+      ));
+      fireEvent.change(screen.getByLabelText('the label'), { target: { value: 'new value' } });
       expect(onChangeSpy.mock.calls.length).toEqual(1);
     });
   });
 
   describe('onFocus API', () => {
     it('sets the onFocus handler', () => {
       const onFocusSpy = jest.fn();
-      const tree = renderer.create((
+      render((
         <Input
           id="foo"
-          label="I am a label for accessibility"
+          label="the label"
           onFocus={onFocusSpy}
         />
-      )).root;
-      tree.findByType('input').props.onFocus();
+      ));
+      fireEvent.focus(screen.getByLabelText('the label'));
       expect(onFocusSpy.mock.calls.length).toEqual(1);
```

Comment:
Not at all related, but I figured this is a good moment to mention it @juanca but I think it will speed up our development if we use test double spys over native jest spys. 

https://github.com/testdouble/testdouble-jest

Not block, nothing you gotta address. Just ping me about it if you have questions

Reply:
Our philosophy is to avoid bringing in too many external dependencies. As of this writing, I haven't seen any benefit from td over spies. 

But if things change, we could do the change in another PR in the future.